src/ngff_tables_prototype/writer.py

```diff
@@ -54,12 +54,23 @@ def write_table_regions(
     instance_key: Optional[str] = None,
 ):
     write_elem(group, table_group_name, adata)
+    group.attrs["tables"] = [table_group_name]
```

Comment:
@will-moore , just to make sure I understand the intention here, this would contain a list of all tables in the group right?

I think it's fine to leave as is for this example, but would this mean that we should actually check to see if there are already tables listed under `tables` and if is so append?

Reply:
Yes, correct. Sorry, we should definitely check if there's already tables and append.

Reply:
No worries, thanks for the clarification!

Reply:
That should handle any existing list of tables: [5017081](https://github.com/kevinyamauchi/ome-ngff-tables-prototype/pull/12/commits/50170810e7d86bf820ffc1fede95bd94bcc71d66)

Reply:
Thanks!